packages/@aws-cdk/aws-kinesisfirehose-destinations/lib/private/helpers.ts

```diff
@@ -64,3 +64,26 @@ export function createLoggingOptions(scope: Construct, props: DestinationLogging
   }
   return undefined;
 }
+
+export function createBufferingHints(
+  bufferingInterval?: cdk.Duration,
+  bufferingSize?: cdk.Size,
+):firehose.CfnDeliveryStream.BufferingHintsProperty | undefined {
+  if (bufferingInterval && bufferingSize) {
+    if (bufferingInterval.toSeconds() < 60 || bufferingInterval.toSeconds() > 900) {
```

Comment:
We seem to be repeating the `bufferingInterval.toSeconds()` and `bufferingSize.toMebibytes()` expressions quite a bit. Maybe some local variables are in order?